SignalMessaging/categories/UIFont+OWS.m

```diff
@@ -122,15 +122,7 @@ - (UIFont *)ows_medium
     // But this doesn't seem to work in practice on iOS 11 using UIFontWeightMedium.
 
     UIFont *derivedFont = [UIFont systemFontOfSize:self.pointSize weight:UIFontWeightMedium];
-
-    if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(11, 0)) {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wpartial-availability"
-        return [[UIFontMetrics defaultMetrics] scaledFontForFont:derivedFont];
```

Comment:
We don't want to use `scaledFontForFont`.

Reply:
Your comment above still says "But this doesn't seem to work in practice on iOS 11 using UIFontWeightMedium."

If that's not the case, could you update the comment? If it is still the case seems like theres more work to do?

Reply:
Yup, `[UIFontDescriptor fontDescriptorByAddingAttributes:...]` doesn't work so we use `[UIFont systemFontOfSize: weight:]` as it stands here.  

I find the docs around `scaledFontForFont` confusing.  I didn't realize that it would affect the font size in some cases.